sql/data-observability-dev/fenix_derived/events_daily_v1/query.sql

```diff
@@ -0,0 +1,155 @@
+-- Generated by ./bqetl generate events_daily
+WITH sample AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    name AS event,
+    category,
+    extra,
+    sample_id,
+    timestamp,
+    metadata,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    client_info.client_id AS client_id,
+    client_info.android_sdk_version AS android_sdk_version,
+    client_info.app_build AS app_build,
+    client_info.app_channel AS app_channel,
+    client_info.app_display_version AS app_display_version,
+    client_info.architecture AS architecture,
+    client_info.device_manufacturer AS device_manufacturer,
+    client_info.device_model AS device_model,
+    client_info.first_run_date AS first_run_date,
+    client_info.telemetry_sdk_build AS telemetry_sdk_build,
+    client_info.locale AS locale,
+    (
+      SELECT
+        ARRAY_AGG(STRUCT(key, value.branch AS value))
+      FROM
+        UNNEST(ping_info.experiments)
+    ) AS experiments
+  FROM
+    org_mozilla_firefox.events e
+  CROSS JOIN
+    UNNEST(e.events) AS event
+  UNION ALL
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    name AS event,
+    category,
+    extra,
+    sample_id,
+    timestamp,
+    metadata,
+    normalized_channel,
+    normalized_os,
+    normalized_os_version,
+    client_info.client_id AS client_id,
+    client_info.android_sdk_version AS android_sdk_version,
+    client_info.app_build AS app_build,
+    client_info.app_channel AS app_channel,
+    client_info.app_display_version AS app_display_version,
+    client_info.architecture AS architecture,
+    client_info.device_manufacturer AS device_manufacturer,
+    client_info.device_model AS device_model,
+    client_info.first_run_date AS first_run_date,
+    client_info.telemetry_sdk_build AS telemetry_sdk_build,
+    client_info.locale AS locale,
+    (
+      SELECT
+        ARRAY_AGG(STRUCT(key, value.branch AS value))
+      FROM
+        UNNEST(ping_info.experiments)
+    ) AS experiments
+  FROM
+    org_mozilla_firefox_beta.events e
```

Comment:
We'll need to fully qualify the table name here since ` org_mozilla_firefox_beta` doesn't exist in the dev project

Reply:
good catch, thanks! 🙇